packages/default/test/renderers/input.control.test.tsx

```diff
@@ -321,3 +321,57 @@ test('validation of nested schema', t => {
   t.is(validation[1].textContent, 'is a required property');
   t.is(validation[2].textContent, 'is a required property');
 });
+test('required field with warning', t => {
```

Comment:
What's the warning? Is is the required marker (i.e. the asterisk)?

Reply:
no idea what this means, I just ported the stuff ;-) . We should rename this.